testcontainers/index.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express"
+import { setupRedis } from "./setupRedis.js"
```

Comment:
Also, shall we get rid of `.js`? Or it's necessary? :thinking:

Reply:
It is, b/c we use mjs Syntax